File: hw_2/Codes/Agcali_AlpTugrul.dart

void main() {
  for(var test = 0; test < 10; test++){
    print("test = $test");
    if(test==3) break;
  }
  
  for( var outer = 0 ; outer < 10; outer++ ) { 
    print("outer = $outer");
    for(var j = 0; j <10; j++){
       if(outer==2) break;      
    }
  }
  print("---------------");
  label:
  for( var outer = 0 ; outer < 10; outer++ ) { 
    print("outer = $outer");
    for(var j = 0; j <10; j++){
       if(outer==2) break label;      
    }
  } 
}

File: hw_2/Codes/Agcali_AlpTugrul.rb
# Online Ruby Compiler (Interpreter)
for test in 1..10 do
 print "test = ",test,"\n"
 if test == 3 
     break 
 end
end

for outer in 1..10 do
    print "outer = ", outer,"\n"
    for j in 1..10 do
        if outer == 3
            break
        end
    end
end

puts "------------"
catch :label do
    for outer in 1..10 do
        print "outer = ", outer,"\n"
        for j in 1..10 do
            throw :label if outer == 3
        end
    end
end
catch :loop do
    i = 0
    print "i = ", i, "\n"
    
    throw :loop if i == 10
end





